discovery/azure/azure.go

```diff
@@ -115,6 +126,15 @@ func validateAuthParam(param, name string) error {
 	return nil
 }
 
+func setCacheParam(param model.Duration, name string) (*cache.Cache[string, *network.Interface], error) {
+	if param.String() == "0s" {
```

Comment:
why using the string method?

Reply:
No sure how to know if the value is 0s I can compare against `model.duration(0* time.Second)`

Reply:
== 0

Reply:
fixed